docs/usage.rst

```diff
@@ -5,12 +5,16 @@
 Usage
 =====
 
-|project_name| consists of a number of commands. Each command accepts
-a number of arguments and options. The following sections will describe each
-command in detail.
+``borg-import`` consists of a number of commands, one for each
+backup system supported. Each accepts a number of arguments and
+options. The following sections will describe each in detail.
 
 General
 -------
 
-...
+.. _rsnapshot:
 
+borg-import rsnapshot
+---------------------
+
+.. generate-usage:: rsnapshot
```

Comment:
With some refinement - mainly better debugging, e.g. dumping the generated rST if it raises any warnings/errors - it should work very nicely.

What I see as an advantage here as well is that headings from the directive are not added to the ToC (although it's possible to hack them in), so all these "Description" ToC entries would go away.